src/Entities/Account.php

```diff
@@ -3,9 +3,6 @@
 
 namespace Choredo\Entities;
 
-use Ramsey\Uuid\UuidInterface;
-use Doctrine\ORM\Mapping as ORM;
```

Comment:
This isn't correct. The UUIDInterface should be pulled in as part of the Constructor (my oversight). 

Will the annotations still work without the second import?

Reply:
Ping here. have you verified that annotations work without that import?